app/templates/components/submissions-repoid-cell.hbs

```diff
@@ -1,26 +1,20 @@
 <div style="min-width:150px">
-{{#each (search-associated 'repositoryCopy' 'publication' (get record 'publication.id')) as |repositoryCopy index|}}
-  {{#if repositoryCopy.externalIds}}
+  {{#each displayIds as |idInfo index|}}
     {{if index ', '}}
-    {{#if repositoryCopy.accessUrl}}
-    <ul class="repoid-cell">
-      {{#each repositoryCopy.externalIds as |externalIds|}}
-        <a href="{{repositoryCopy.accessUrl}}" target="_blank">
-          <li title="{{externalIds}}">{{externalIds}}</li>
-        </a>
-      {{/each}}
-    </ul>
-    {{else}}
+    {{#if idInfo.url}}
       <ul class="repoid-cell">
-        {{#each repositoryCopy.externalIds as |externalIds|}}
-          <li title="{{externalIds}}">{{externalIds}}</li>
+        {{#each idInfo.ids as |id|}}
+          <a href="{{url}}" target="_blank">
```

Comment:
This is showing up blank for some reason - so the anchor tag is there but the url is empty.